### PR TITLE
Minikube-based development environment

### PR DESCRIPTION
**Pull request by shanemcd**
Works in conjunction with https://github.com/ansible/awx-operator/pull/71

See docs/development/minikube.md

### CODE REVIEW

**Comment by shanemcd** [comment]
recheck

**Comment by shanemcd** [comment]
recheck

**Comment by shanemcd** [comment]
recheck

File: docs/development/minikube.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+## Deploy the AWX Operator
+
+Clone the [awx-operator](https://github.com/ansible/awx-operator).
+
+If you are not changing any code in the operator itself, simply run:
+
+```
+$ ansible-playbook ansible/deploy-operator.yml
```
Comment (ryanpetrello) [inline docs/development/minikube.md:41]: Might be worth mentioning that you need a few Python dependencies in addition to ansible, namely `pip install openshift`.

Comment (shanemcd) [reply docs/development/minikube.md:41]: Good call. Done.

**Review by ryanpetrello — APPROVED** [review]
I gave this a whirl locally w/ a `devel` build and it looks `<chefs-kiss>`:

```
~ kubectl get pods
NAME                           READY   STATUS    RESTARTS   AGE
awx-7b7848b8b5-ts4r4           3/3     Running   0          26m
awx-operator-7c78bfbfd-ln29k   1/1     Running   1          27m
awx-postgres-0                 1/1     Running   0          26m
~ kubectl exec awx-7b7848b8b5-ts4r4 -c awx-task -- awx-manage --version
16.0.0
~ curl -sk `minikube service awx-service --url`/api/v2/ping/ | jq
{
  "ha": false,
  "version": "16.0.0",
  "active_node": "awx-7b7848b8b5-ts4r4",
  "install_uuid": "0ef8f303-074d-4a95-99e6-7639ac972d3e",
  "instances": [
    {
      "node": "awx-7b7848b8b5-ts4r4",
      "uuid": "00000000-0000-0000-0000-000000000000",
      "heartbeat": "2021-01-26T20:48:43.068034Z",
      "capacity": 277,
      "version": "16.0.0"
    }
  ],
  "instance_groups": [
    {
      "name": "tower",
      "capacity": 277,
      "instances": [
        "awx-7b7848b8b5-ts4r4"
      ]
    }
  ]
}
```

File: Makefile [hunk 2]
```diff
@@ -267,11 +267,27 @@ collectstatic:
...
+	    --vacuum \
+	    --processes=5 \
+	    --harakiri=120 --master \
+	    --no-orphans \
+	    --py-autoreload 1 \
+	    --max-requests=1000 \
+	    --stats /tmp/stats.socket \
+	    --lazy-apps \
+	    --logformat "%(addr) %(method) %(uri) - %(proto) %(status)" \
+	    --hook-accepting1="exec: $(UWSGI_DEV_RELOAD_COMMAND)"
```
Comment (ryanpetrello) [inline Makefile:290]: This is a little easier on the eyes 😂

**Pull request merged**